firmware_io/simulink_input.csv

```diff
@@ -1,2 +1,51 @@
+DI_AccelPedalPos,float,%,0,1,Accelerator pedal position,0 is released,1 is fully pressed
+DI_BrakePedalPos,float,%,0,1,Brake pedal position,0 is released,1 is fully pressed
+DI_StartButton,bool,,0,1,Start button on dash (active low),0 is released,1 is pressed
+VD_FrtWheelSpd,int,rpm,-2000,2000,Average wheel speed of the front wheels,,
+BM_HvilFeedback,bool,,0,1,HVIL status,0 is open,1 is closed
+BM_prechrgContactorSts,bool,,0,1,Precharge contactor status,0 is open,1 is closed
+BM_HVposContactorSts,bool,,0,1,HV positive contactor status,0 is open,1 is closed
+BM_HVnegContactorSts,bool,,0,1,HV negative contactor status,0 is open,1 is closed
+BM_PackSOC,uint,%,0,100,HV battery state of charge,0 is empty,100 is full
+BM_LowThermValue,int,C,-128,127,Lowest thermistor value reported by TMS,,
+BM_HighThermValue,int,C,-128,127,Highest thermistor value reported by TMS,,
+BM_AvgThermValue,int,C,-128,127,Average thermistor value reported by TMS,,
+VD_AccelerationX,float,m/s^2,-64,63.875,X acceleration reported by IMU,X acceleration with a resolution of 0.125 m/s^2,
+VD_AccelerationY,float,m/s^2,-64,63.875,Y acceleration reported by IMU,Y acceleration with a resolution of 0.125 m/s^2,
+VD_AccelerationZ,float,m/s^2,-64,63.875,Z acceleration reported by IMU,Z acceleration with a resolution of 0.125 m/s^2,
+VD_AngularRateX,float,deg/s,-256,255.75,Roll velocity,X angular rate with a resolution of 0.25 deg/s,
+VD_AngularRateY,float,deg/s,-256,255.75,Pitch velocity,Y angular rate with a resolution of 0.25 deg/s,
+VD_AngularRateZ,float,deg/s,-256,255.75,Yaw velocity,Z angular rate with a resolution of 0.25 deg/s,
+VD_ImuValid,bool,,0,1,IMU status signal,0 - IMU invalid,1 - IMU valid
+AMK0_bSystemReady,bool,,0,1,System Ready ,,
+AMK0_bError,bool,,0,1,Error,,
+AMK0_bWarn,bool,,0,1,Warning,,
+AMK0_bQuitDcOn,bool,,0,1,HV activation acknloedgement,,
+AMK0_bDcOn,bool,,0,1,HV activation level,,
+AMK0_bQuitInverterOn,bool,,0,1,Controller enable acknowledgment,,
+AMK0_bInverterOn,bool,,0,1,Controller enable level,,
+AMK0_bDerating,bool,,0,1,Derating (torque limitation active),,
+AMK0_ActualVelocity,int,rpm,-32768,32767,,,
+AMK0_TorqueCurrent,int,,-32768,32767,,,
+AMK0_MagnetizingCurrent,int,,-32768,32767,,,
+AMK0_TempMotor,int,0.1 C,-3276,3276,Motor temperature,,
+AMK0_TempInverter,int,0.1 C,-3276,3276,Cold plate temperature,,
+AMK0_ErrorInfo,uint,,0,65535,CAN diagnostic message from inverter,,
+AMK0_TempIGBT,int,0.1 C,-3276,3276,IGBT temperature,,
+AMK1_bSystemReady,bool,,0,1,System Ready ,,
+AMK1_bError,bool,,0,1,Error,,
+AMK1_bWarn,bool,,0,1,Warning,,
+AMK1_bQuitDcOn,bool,,0,1,HV activation acknloedgement,,
+AMK1_bDcOn,bool,,0,1,HV activation level,,
+AMK1_bQuitInverterOn,bool,,0,1,Controller enable acknowledgment,,
+AMK1_bInverterOn,bool,,0,1,Controller enable level,,
+AMK1_bDerating,bool,,0,1,Derating (torque limitation active),,
+AMK1_ActualVelocity,int,rpm,-32768,32767,,,
+AMK1_TorqueCurrent,int,,-32768,32767,,,
```

Comment:
please fill in units (ex amperes, milli-amps)

Reply:
These are CAN signals

Reply:
This signal is unit less, please refer to page 61 on [AMK Datasheet](https://mcmasteru365.sharepoint.com/sites/MACFormulaElectric686/Shared%20Documents/ENG%20-%20Software%20and%20Vehicle%20Controls/AMK_Documentation/AMK_Motor_Datasheet_2020_44.pdf?CT=1710967117689&OR=ItemsView)